boards/arm/pan1783_evb/pan1783_evb_cpuapp_defconfig

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF53X=y
+CONFIG_SOC_NRF5340_CPUAPP_QKAA=y
+CONFIG_BOARD_PAN1783_EVB_CPUAPP=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable TrustZone-M
+CONFIG_ARM_TRUSTZONE_M=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# Enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# clock config
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC=y
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL=n
+
+CONFIG_PINCTRL=y
```

Comment:
no longer needed, it's already selected by drivers

Reply:
Okay, could you maybe describe that? All of them can be removed? Or the whole file? 

I've tried to remove some of the configurations and tried to build samples/hello_world, but get build errors now or sample is not working (no uart output, clock is not working,...)? 

Sorry I'm still not that familiar with these topics. Thanks.